### PR TITLE
Added route APIs in scanner group

### PR DESCRIPTION
**Pull request by tushar-balwani**
# Description

Implemented route APIs `list_routes` and `edit_routes` in `scanner-groups` package as per documentation

Fixes # (issue)

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

# How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

- [x] Tested by listing routes associated to scanner-group
```
routes = api.scanner_groups.list_routes(1)
pprint(routes)
```
- [x] Tested by updating routes associated to scanner-group
```
tio.scanner_groups.edit_routes(1, ['127.0.0.1'])
```
- [x] Added below test-cases for list and edit routes
`test_edit_routes_in_scanner_group_invalidinputerror`
`test_edit_routes_in_scanner_group_typeerror`
`test_edit_routes_in_scanner_group_success`
`test_list_routes_in_scanner_group_success`

**Test Configuration**:
* Python Version(s) Tested: 3.8.6
* Tenable.sc version (if necessary):

# Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

**Review by levwais — COMMENTED** [review]
lgtm

**Review by valliammal — COMMENTED** [review]
tests/io/test_scanner_groups.py 
Wildcard import is not permitted.  Functions need to have comments before that with docstring

**Review by valliammal — APPROVED** [review]
Changes are approved , code looks good.

File: tenable/io/scanner_groups.py [hunk 1]
```diff
@@ -71,14 +74,14 @@ def create(self, name, group_type=None):
                 default='load_balancing', choices=['load_balancing'])
         }).json()
 
-    def delete(self, id):
+    def delete(self, group_id):
```
Comment (valliammal) [inline tenable/io/scanner_groups.py:77]: Add some doc string above the functions , for all the functions

**Review by levwais — APPROVED** [review]

**Pull request merged**